package.json

```diff
@@ -25,6 +25,7 @@
     "@headlessui/vue": "^1.7.16",
     "@popperjs/core": "^2.11.2",
     "@redis/client": "^1.5.8",
+    "@sentry/browser": "^7.15.0",
```

Comment:
Just curious since this version is over a year old: is this being pinned for compatibility?

Reply:
Hmm... @akhilnarang we should pick highest current supported version. Not sure if our deployment is compatible though.

Reply:
Anyway `^` means it will install latest minor?

So it seems correct only: https://www.npmjs.com/package/@sentry/browser 

See lock file.

Reply:
> Not sure if our deployment is compatible though.

Most things should be fine since even latest python SDK is fine, besides the extra field being sent that's getting discarded for now.

Will bump this before merge, for now its older since I had just rebased the old branch.